pyramid_tm/__init__.py

```diff
@@ -94,7 +94,7 @@ def tm_tween(request):
                 try:
                     t.note(text_(request.path_info))
                 except UnicodeDecodeError:
-                    t.note("Unable to decode path as unicode")
+                    t.note(text_("Unable to decode path as unicode"))
```

Comment:
Oh sorry I just woke up.... This is fine, no need to update it. I thought this commit was from my previous attempt at fixing the issues.